README.md

```diff
@@ -63,43 +63,23 @@ The CronJob calls the endpoint `/prisoner-index/queue-housekeeping` which is not
 
 ### Running
 
-`localstack` is used to emulate the AWS SQS and Elastic Search service. Any commands in `localstack/setup-es.sh` will be run when `localstack` starts, so this contains commands to create the appropriate queues. Localstack listens on port 4566.
```

Comment:
a can imagine you being annoyed at whatever this long text means and ripping it out :-)